apps/client/pages/onboarding.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import type { Story, Meta } from '@storybook/react'
```

Comment:
Could you move this to the new /stories directory? Having the page stories in client/pages does some wonky stuff with the build process since it's expecting everything there to be a route and page.